src/bootstrap/Cargo.lock

```diff
@@ -271,8 +274,8 @@ dependencies = [
  "aho-corasick",
  "bstr",
  "log",
- "regex-automata",
- "regex-syntax",
+ "regex-automata 0.4.9",
+ "regex-syntax 0.8.5",
```

Comment:
Seems like we duplicated few dependencies. Is it possible to avoid this?

Reply:
Probably, I may have used `tracing*` versions that are too new. I'll double check.

Reply:
Fixed by not specifying newer minor versions.

Reply:
This one is duplicated because

```
├── tracing-subscriber v0.3.19
│   ├── matchers v0.1.0
│   │   └── regex-automata v0.1.10
│   │       └── regex-syntax v0.6.29
```

I don't think this can be dedup'd because it's a transitive dep of `tracing-subscriber` when `env-filter` feature is enabled (which is used to filter like `BOOTSTRAP_TRACING=bootstrap::core::build_steps::test=TRACE`).

Reply:
`tracing` stuck with updating `matchers` almost half year :-(

Reply:
Want to cargo have ability to simply apply patches to dependencies, like `[patch]` but with diff files.

Reply:
Oh hey... Chris opened a PR for `nu-ansi-term` at https://github.com/tokio-rs/tracing/pull/3049 and the `matchers` bump is stuck on maintainer bandwidth probably at https://github.com/tokio-rs/tracing/pull/3033